SourceKitStressTester/Sources/StressTester/ActionGenerators.swift

```diff
@@ -51,7 +51,8 @@ extension ActionGenerator {
     for frontAction in actionToken.frontActions {
       switch frontAction {
       case .codeComplete:
-        actions.append(.codeComplete(offset: contentStart))
+        let expected: ExpectedResult? = withReplaceTexts ? nil : actionToken.frontExpectedResult
+        actions.append(.codeComplete(offset: contentStart, expectedResult: expected))
```

Comment:
The stress tester has a few different modes for generating SourceKit requests. Some of them modify the original source file via replaceText requests (to test invalid/incomplete code) and some of them leave it in its original compiling state. We only want to add an expected result for the latter modes, because if the file is being modified there's no guarantee the symbol associated with this token is actually a valid completion anymore (e.g. maybe the base type of a member completion became an error type because of the modifications).

Reply:
Makes sense. Thanks!